core/unit_test/incremental/Test12a_ThreadScratch.hpp

```diff
@@ -118,11 +118,14 @@ struct ThreadScratch {
   }
 };
 
+// FIXME_SYCL unclear if it can be implemented with out SYCL2020
+#ifndef KOKKOS_ENABLE_SYCL
```

Comment:
Is that the only way to disable test 12a?

Reply:
I think so. 
The above comment is also not true with this version. I just wanted to confirm that I get the same functionality as in the pull request corresponding to the current TeamPolicy implementation. 
IMHO, it's fine without fixing it here but I'm definitely working on it.

Reply:
I updated the comment.